docs/ops/movement/ScatterUpdate_3.md

```diff
@@ -7,49 +7,45 @@
 **Short description**: *ScatterUpdate* creates a copy of the first input tensor with updated elements specified with second and third input tensors.
 
 **Detailed description**: *ScatterUpdate* creates a copy of the first input tensor with updated elements in positions specified with `indices` input
-and values specified with `updates` tensor starting from the dimension with index `axis`. For the `data` tensor of shape `[d_0, d_1, ..., d_n]`,
-`indices` tensor of shape `[i_0, i_1, ..., i_k]` and `updates` tensor of shape
-`[d_0, d_1, ... d_(axis - 1), i_0, i_1, ..., i_k, d_(axis + 1), ..., d_n]` the operation computes
+and values specified with `updates` tensor starting from the dimension with index `axis`. For the `data` tensor of shape \f$[d_0, d_1, \dots, d_n]\f$,
+`indices` tensor of shape \f$[i_0, i_1, \dots, i_k]\f$ and `updates` tensor of shape
+\f$[d_0, d_1, \dots, d_{axis - 1}, i_0, i_1, \dots, i_k, d_{axis + 1}, \dots, d_n]\f$ the operation computes
 for each `m, n, ..., p` of the `indices` tensor indices:
 
-```
-data[..., indices[m, n, ..., p], ...] = updates[..., m, n, ..., p, ...]
-```
 
-where first `...` in the `data` corresponds to first `axis` dimensions, last `...` in the `data` corresponds to the
+\f[data[\dots,\;indices[m,\;n,\;\dots,\;p],\;\dots] = updates[\dots,\;m,\;n,\;\dots,\;p,\;\dots]\f]
+
+where first \f$\dots\f$ in the `data` corresponds to first `axis` dimensions, last\f$\dots\f$ in the `data` corresponds to the
```

Comment:
It's quiet hard to understand this sentence.. "to first `axis` dimensions".
What does it mean?